client/src/components/pages/QuizSend.js

```diff
@@ -125,7 +125,7 @@ function QuizSend() {
 						alert(
 							"Emails have been sent successfully. You will be redirected to dashboard."
 						);
-						navigate("/quizlist");
+						navigate("/");
```

Comment:
Did we want to leave it as going back to dashboard or going back to quizlist?

Reply:
ah okay I guess it got switched back. The alert is going back to dashboard, so I guess it's okay that way? Feel free to change it if you want.